cfme/tests/physical_infrastructure/ui/test_physical_server_list_buttons.py

```diff
@@ -0,0 +1,93 @@
+# -*- coding: utf-8 -*-
+import pytest
+from selenium.common.exceptions import StaleElementReferenceException
+
+from cfme.physical.provider.lenovo import LenovoProvider
+from cfme.utils.appliance.implementations.ui import navigate_to
+from cfme.common.physical_server_views import (
+    PhysicalServersView,
+)
+from cfme.utils.wait import wait_for
+
+pytestmark = [pytest.mark.tier(3), pytest.mark.provider([LenovoProvider], scope="module")]
+
+
+@pytest.fixture(scope="module")
+def physical_servers_collection(appliance, setup_provider_modscope):
```

Comment:
provider scopes should always be function, we're removing provider modscope.

The setup_provider fixture will handle only adding/removing the provider when necessary.

This shouldn't be a fixture, there's no setup. Just use the appliance fixture locally in your tests to access collections.

Reply:
ok, I changed the scope and I set as a function, but I added the setup_provider in a another fixture with the same scope(function) because I think that we need ensure that always exist a provider inside the manageIQ before we run any test. Also I think that we most be able to running a specific test without erros.